tests/app/UnitTests/GitCommands.Tests/Git/AheadBehindDataProviderTests.cs

```diff
@@ -74,6 +74,7 @@ public void GetData_should_return_null_result_if_unable_match_branch(string resu
 
         [TestCase("::::::::my-branch")]
         [TestCase("::gone::::::branch-with-no-more-remote")]
+        [TestCase("gone::::refs/remotes/origin/callbacks::::new-branch")]
```

Comment:
❔
```suggestion
        [TestCase("gone::::refs/remotes/origin/renamed-branch::::new-branch-with-push-spec-to-same-repo")]
```

Reply:
(I agree to your other comments.)

Here, my point is: Testcases use to illustrate how the code is expected to work. The idea behind "callbacks" vs. "new-branch" is not obvious. One (me) needs at least the PR description in order to get what _could_ be meant.

Reply:
➡️ Tweaked the test and added a comment